code/modules/clothing/suits/modular_armor.dm

```diff
@@ -207,6 +221,39 @@
 	item_state = "plate_blue"
 	slowdown = 0 // the SCC is hacking
 
+/obj/item/clothing/accessory/armor_plate/tcaf
+	name = "tcaf legionnaire carapace"
```

Comment:
in some places you've got tcaf and in others TCAF, wonder if it'd be better to have them consistant

Reply:
That's primarily based off of how the TCFL items were labeled in their respective codes. Sometimes the names had all caps, sometimes it was all lowercase, so I just matched what they had incase there was some weird code reason I didn't understand. It wouldn't be too hard though to make them all lowercase, but if I understand correctly, we'll need to undo the changes made earlier to make them improper, and those edits were made to both TCFL and TCAF gear.